openedx/core/djangoapps/user_api/preferences/tests/test_api.py

```diff
@@ -131,25 +111,19 @@ def test_change_email_optin(self, age, option, second_option, expected_result):
         profile.year_of_birth = year_of_birth
         profile.save()
 
-        profile_api.update_email_opt_in(user, course.id.org, option)
-        profile_api.update_email_opt_in(user, course.id.org, second_option)
+        update_email_opt_in(user, course.id.org, option)
+        update_email_opt_in(user, course.id.org, second_option)
 
         result_obj = UserOrgTag.objects.get(user=user, org=course.id.org, key='email-optin')
         self.assertEqual(result_obj.value, expected_result)
 
-    @raises(profile_api.ProfileUserNotFound)
-    def test_retrieve_and_update_preference_info_no_user(self):
```

Comment:
@andy-armstrong Also make sure that the expected errors around non-existing users are covered.